packages/accordions/src/styled/timeline/StyledItemIcon.ts

```diff
@@ -29,12 +29,12 @@ export const StyledItemIcon = styled(StyledBaseIcon).attrs({
 })<IStyledItemIcon>`
   z-index: 1;
   box-sizing: content-box;
-  background-color: ${props =>
-    props.$surfaceColor || getColorV8('background', 600 /* default shade */, props.theme)};
+  background-color: ${({ $surfaceColor, theme }) =>
+    $surfaceColor || getColor({ theme, variable: 'background.default' })};
   padding: ${props => props.theme.space.base}px 0;
   width: ${props => math(`${props.theme.iconSizes.sm} + 1`)}; /* [1] */
   height: ${props => math(`${props.theme.iconSizes.sm} + 1`)}; /* [1] */
-  color: ${props => getColorV8('neutralHue', 600, props.theme)};
+  color: ${({ theme }) => `${getColor({ theme, variable: 'border.emphasis' })}`};
```

Comment:
With more than one color property, it is probably worth it to move out to a dedicated `colorStyles` function where the CSS is more readable/maintainable.